adapters/Chartboost/Public/Headers/GADMChartboostExtras.h

```diff
@@ -1,29 +0,0 @@
-// Copyright 2016 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#import <Foundation/Foundation.h>
-#import <GoogleMobileAds/GoogleMobileAds.h>
-
-typedef NS_ENUM(NSUInteger, CBFramework);
-
-/// Keys for the Chartboost extra assets.
-@interface GADMChartboostExtras : NSObject<GADAdNetworkExtras>
-
-/// Chartboost custom framework.
-@property(nonatomic, assign) CBFramework framework;
-
-/// Chartboost custom framework version.
-@property(nonatomic, copy) NSString *frameworkVersion;
```

Comment:
Since this is now deleted, how can the publishers pass the parameters as in https://developers.google.com/admob/ios/mediation/chartboost#network-specific_parameters?

Reply:
SDK does not use these parameters anymore.